sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -177,8 +178,21 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
       return
     }
     mapboxRenderer.onDrawFrame()
+    // assuming render event queue holds user's runnables with OpenGL ES commands
+    // it makes sense to execute them after drawing a map but before swapping buffers
+    if (renderEventQueue.isNotEmpty()) {
```

Comment:
IMHO drainQueue still looked better(

Reply:
we could not re-use one method now for draining queue for snapshot events (Runnable) and draining a queue for custom class (RenderEvent with nullable runnable field)

Reply:
as we can't re-use it and taking into account this logic is called once in code - I see no sense in leaving that method

Reply:
Ok, haven't realized the difference